CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 
+v4.5.0
```

Comment:
not sure about the version number. Is it v4.5.0 or v4.4.2?

Reply:
4.5.0 should be fine, as it is not a bug fix and not a breaking change. You also need to bump the version in package.json

Reply:
thanks!